src/ScenarioPreview/ScenarioPreview.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import { Meta, StoryFn } from "@storybook/react";
+
+import React from "react";
+import { ScenarioPreview } from "./ScenarioPreview";
+import { ScenarioPreviewProps } from "./ScenarioPreview.types";
+
+const meta: Meta<typeof ScenarioPreview> = {
+  component: ScenarioPreview,
+  title: "Preview/ScenarioPreview"
+};
+export default meta;
+
+// Story Template
+const Template: StoryFn<ScenarioPreviewProps> = args => {
+  return <ScenarioPreview {...args} />;
+};
+
+/**
+ * Default story
+ */
+export const Default = Template.bind({});
+Default.args = {
+  createdAt: "23-06-06 08:09:24",
+  description: "Small Description",
+  file: "RoadName",
+  format: "CarMaker",
+  formatVersion: "11.1",
+  href: "https://example.com",
+  image: "https://picsum.photos/id/191/400/200",
+  name: "CityDriving",
+  roadLabel: "RoadName", // Updated to use Road Label
+  sx: {
+    borderRadius: "8px",
+    boxShadow: 3,
+    maxWidth: "480px",
+    padding: "16px"
```

Comment:
For multiples of 8 it might be a good idea to use the MUI system, incase we ever change the increment.

```suggestion
  sx: {
    borderRadius: 1,
    boxShadow: 3,
    maxWidth: "480px",
    padding: 2
```